tutorials/beginner/decoupling_logic_and_execution.ipynb

```diff
@@ -6,25 +6,25 @@
    "source": [
```

Comment:
<p>FugueWorkflow I think should not have the (). Either way we should make the two appearances here consistent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
done